### DISCUSSION TITLE
首次绑定（购买后如何绑定高级账户）

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1573] [高级账户]
> 当获取 **兑换码** 后，在 **浏览器→ 选项页（浏览器右上角简悦图标，点击右键，偏好设置 → 右上角 → 账户 icon → 绑定你的高级账户资格**

[![BqqqDP.png](https://s1.ax1x.com/2020/11/11/BXR1xK.png)](https://imgchr.com/i/BqqqDP)

![image](https://user-images.githubusercontent.com/81074/131622383-e21601c7-9d70-452a-b145-69be93586406.png)

![image](https://user-images.githubusercontent.com/81074/131622611-a8ca19f2-9d3d-4884-9231-5fc7eb4bc560.png)

![image](https://user-images.githubusercontent.com/81074/141410481-baafa046-46cf-480f-8b32-e94872edebd4.png)

## 当成功绑定后仍出现加锁的情况

![image](https://user-images.githubusercontent.com/81074/131622753-c7d305ba-4b75-472c-b477-9603c2ffc53c.png)

1.  在 **浏览器→ 选项页（浏览器右上角简悦图标，点击右键，偏好设置 → 右上角 → 账户 icon → 解除对当前设备的绑定**

![image](https://user-images.githubusercontent.com/81074/131622869-d2f09c1a-e122-4092-8ea3-4f6a1624d405.png)

2. 重启浏览器

3. 在 **浏览器→ 选项页（浏览器右上角简悦图标，点击右键，偏好设置 → 右上角 → 账户 icon → 仅授权当前设备**

![image](https://user-images.githubusercontent.com/81074/131622958-87916ca3-0a91-49eb-a2fb-0aafb38baf6b.png)

4. 当出现下图所示的提示后，**重启浏览器即可**。

![image](https://user-images.githubusercontent.com/81074/131623059-1f5941c8-92d6-4386-928b-195ed1e23684.png)

## 关联阅读

- [绑定高级账户](https://github.com/Kenshin/simpread/discussions/1573)

- [在新设备上登录](https://github.com/Kenshin/simpread/discussions/1574)

- [使用授权登录](https://github.com/Kenshin/simpread/discussions/1570)

- [取消授权](https://github.com/Kenshin/simpread/discussions/1575)

- [重新授权](https://github.com/Kenshin/simpread/discussions/1576)

- [找回丢失的 UID （高级账户资格）](https://github.com/Kenshin/simpread/discussions/1571)

- [找回 UID 后如何恢复高级账户资格](https://github.com/Kenshin/simpread/discussions/1958)

- [重新找到 License（兑换码）](https://github.com/Kenshin/simpread/discussions/2756)

- [安装和配置同步助手](https://github.com/Kenshin/simpread/discussions/2754)

**Comment by dosu15** [comment]
我首次购买，点绑定"您的高级帐户资格"什么反应都没有？

**Reply by Kenshin** [reply]
嗨，确实也没有遇到过这个问题。看看是否是 https://github.com/Kenshin/simpread/discussions/1567 造成的？
另外，如果没有稍后读并且是新用户的话，重装简悦试试。

**Comment by hqweay** [comment]
<img width="846" alt="image" src="https://user-images.githubusercontent.com/23608642/141301999-c887812a-6afc-4251-ab65-b7a93877c55a.png">

这是不是想说「务必保存好」？- -

**Reply by Kenshin** [reply]
哈哈，是的… 明天改下这里。😂

**Reply by Kenshin** [reply]
已修复。

🙏

**Comment by gzqd** [comment]
绑定一直出现：“后台服务器出现了问题，点击重试(也可能是设置造成的问题，请查看Github Issues置顶)”这个问题，请问是什么原因呢？

**Reply by Kenshin** [reply]
因为你没有给简悦相应的权限，解决方案 https://www.yuque.com/kenshin/simpread/tmmwhl

**Comment by killus-tianyi** [comment]
我购买时使用的是手机号，没有获得兑换码怎么办

**Reply by Kenshin** [reply]
什么渠道购买的？

**Comment by wushimin** [comment]
在软件提示高级账户到期前12天购买了新的license。是否是要等旧的license到期之后再绑定新的？

**Reply by Kenshin** [reply]
简悦是买断制，所以我没明白你的问题是... 😂

**Reply by wushimin** [reply]
不好意思，你这么一提醒，我查了下是我的quicker会员快到期了，之前没注意😂能退吗或者转给其他人？

**Reply by wushimin** [reply]
主要是怕license浪费了，可以当捐助或者送给某位还没升级的用户吧

**Reply by Kenshin** [reply]
哈哈，小事情，你可以先留着，等更多设备上线后再使用。

**Comment by dabaicaiyikuai** [comment]
购买的时候把UID填错了怎么办呢？

**Reply by Kenshin** [reply]
没事，以首次升级到高级账户时的 UID 为准。（首次升级后会自动下载一个文件，请勿遗失）
